### ISSUE TITLE
Participants waiting to be accepted in room do not enter when individually accepted

### ISSUE THREAD

**Issue by pielonet** [bigbluebutton/bigbluebutton#10906]
**Describe the bug**
Setup a room whith moderated access.
When participants try to join the room whith an invitation link they receive a message telling them to wait for acceptance.
The moderator receives a notification asking to "Accept" or "Deny" the user.
When the moderator clicks on "Accept" nothing happens : the user keeps waiting ans is not automatically joined to the room.

**Expected behavior**
The user should automatically join the room when accepted.

**Screenshots**
![Capture d’écran_2020-11-26_11-58-53](https://user-images.githubusercontent.com/15911394/100343102-3a76aa80-2fdf-11eb-9b94-296b3fbf0323.png)
![Capture d’écran_2020-11-26_11-58-21](https://user-images.githubusercontent.com/15911394/100343151-48c4c680-2fdf-11eb-9e96-1665cd37cfb2.png)

**BBB version (optional):**
BigBlueButton continually evolves. Providing the version/build helps us to pinpoint when an issue was introduced.
Example:
$ sudo bbb-conf --check | grep BigBlueButton
BigBlueButton Server 2.2.29 (2244)

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser Firefox
 - Version 83

**Comment by erisd** [comment]
Do you see a broken websocket connection in your browser while waiting?

**Comment by pielonet** [comment]
I don't see a broken websocket. Here is a screenshot of the network inspection from Firefox when opening url /client/guest-wait.html?sessionToken=alycdpsktoaiheht
![Capture d’écran_2020-11-26_17-39-17](https://user-images.githubusercontent.com/15911394/100375715-84c35000-300e-11eb-8506-3c1a30678dcc.png)

**Comment by pielonet** [comment]
Actually, there is no websocket at all.
Same bug with 2.2.28
Works in 2.2.26
Not tested in 2.2.27

**Comment by pedrobmarin** [comment]
> Do you see a broken websocket connection in your browser while waiting?

This doesn't make any sense since the waiting page does not rely any kind of permanent connection

**Comment by pielonet** [comment]
Fred Dixon help me out in solving this bug I had introduced myself

In file /var/www/bigbluebutton/client/guest-wait.html there is a configuration variable called "TIMEOUT_MESSAGE_HERE". I had replaced this text with a French equivalent containing a single quote. As the single quote was not escaped it broke the Javascript code.
Here is the code corrected :+1: 

```
        if (attempt >= limit) {
          updateMessage('Temps d\'attente dépassé. Veuillez recharger la page.');
          return;
        }
```
As en enhancement it would be better to be able to change this text in a configuration file rather than directly in the Javascript code.
